upload/admin/model/extension/smailyforopencart/admin.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+require_once(DIR_SYSTEM . 'library/smailyforopencart/request.php');
```

Comment:
I would have request class `require_once` in every file that uses `SmailyForOpenCart\Request` class. For just in case, because OpenCart can execute code however it likes, and it might not be through entrypoint which includes `request.php`.

The beauty of `require_once` is that it won't include the file, if it has already been required, so it is safe to use multiple times.

Reply:
Yes, of course. Added in commit 7c07825